cpp/src/io/shp/polygon_shapefile_reader.cpp

```diff
@@ -141,6 +150,7 @@ read_polygon_shapefile(std::string const& filename)
   std::vector<double> xs;
   std::vector<double> ys;
 
+  _reverse_winding = reversed;
```

Comment:
Remove global variable. Plumb the parameter down through the internal functions.

Reply:
But it is so many! Isn't there a neater way? :)

Reply:
read_polygon_shapefile, read_layer, read_ring. That's not many in my book.

Reply:
:+1: